lib/ohai.rb

```diff
@@ -17,6 +17,5 @@
 #
 
 require "ohai/version"
-require "ohai/config"
```

Comment:
require what you use. Implicit requirements have caused us so many bugs over the years.